UnitTests/HasherTests.m

```diff
@@ -1,8 +1,13 @@
+#ifndef MPARTICLE_LOCATION_DISABLE
+@import mParticle_Apple_SDK;
+#else
+@import mParticle_Apple_SDK_NoLocation;
+#endif
+
 #import <XCTest/XCTest.h>
 #import "MPEnums.h"
 #import "MPEvent.h"
 #import "MPBaseTestCase.h"
-#import "MPIHasher.h"
 #import "mparticle.h"
```

Comment:
if you're changing to module import couldn't you also remove MPEvent.h, mparticle.h, and MPEnums.h?

Reply:
Good point, any public headers should not need to be imported anymore, though this is just a unit test so doesn't really matter. I'll see if I can clean it up real quick.

Reply:
actually I'm going to close this and re-open a new one for the real PR. I'll include this change.